src/main/java/core/basesyntax/service/impl/ReadFromFileServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReadFromFileService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+public class ReadFromFileServiceImpl implements ReadFromFileService {
+    private static final String END_LINE = System.lineSeparator();
+
+    @Override
+    public String readFromFile(String path) {
+        String line = "";
+        StringBuilder dataFromFile = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(path))) {
```

Comment:
A little lifehack - Files.readString(Path.of(path)). The Files class has many methods that can make your life better

Reply:
thanks. Very useful lifehack